packages/react-searchbox/src/components/SearchBox.js

```diff
@@ -534,17 +537,31 @@ class SearchBox extends React.Component {
                       </li>
                     ))}
                     {!currentValue
-                      ? recentSearches.map((sugg, index) => (
+                      ? (recentSearches || []).map((sugg, index) => (
                           <li
                             {...getItemProps({ item: sugg })}
                             key={`${index + 1}-${sugg.value}`}
                             style={{
                               backgroundColor: this.getBackgroundColor(
                                 highlightedIndex,
                                 index
-                              )
+                              ),
+                              justifyContent: 'flex-start'
                             }}
                           >
+                            <div style={{ padding: '0 10px 0 0' }}>
+                              <CustomSvg
+                                iconId={`${sugg.id}-icon`}
```

Comment:
@anik-ghosh-au7 There is no `id` field present in the suggestion. How did you even test it?
Can you follow the code or log the suggestion object or just check the docs to find the suggestion object schema.